lib/api/formtools/grid.js

```diff
@@ -0,0 +1,263 @@
+var linz = require('../../../');
+    async = require('async'),
+    clone = require('clone');
+
+(function  () {
+
+    function getSettings(modelName, cb) {
+
+        if (!modelName || typeof(modelName) !== 'string' || !modelName.length) {
+            throw new Error('modelName is required and must be of type string and cannot be emptied.');
```

Comment:
Should read "modelName is required and must be of type String and cannot be empty."